tests/js/openURLsTests.js

```diff
@@ -151,29 +151,29 @@
     const handlestRequestTestCases = [{
         name: "Open same tab",
         details: {
-            url: "https://opensametab.morphic.org/actual.org/url"
+            url: "https://opensametab.morphic.org/redirect/https%3A%2F%2Factual.org/url"
         },
         expected: {
             response: {cancel: true},
-            args: ["https://actual.org/url", false]
+            args: [new URL("https://actual.org/url"), false]
         }
     }, {
         name: "Refresh tab",
         details: {
-            url: "http://refreshsametab.morphic.org/actual.org/url"
+            url: "http://refreshsametab.morphic.org/redirect/http%3A%2F%2Factual.org/url"
         },
         expected: {
             response: {cancel: true},
-            args: ["http://actual.org/url", true]
+            args: [new URL("http://actual.org/url"), true]
         }
     }, {
         name: "Unfiltered URL",
         details: {
-            url: "http://morphic.org/actual.org/url"
+            url: "http://morphic.org/redirect/https%3A%2F%2Factual.org/url"
         },
         expected: {
             response: {cancel: true},
-            args: ["http://morphic.org/actual.org/url", false]
+            args: [new URL("http://morphic.org/redirect/https%3A%2F%2Factual.org/url"), false]
         }
```

Comment:
Need test cases for when the response is `{cancel: false}`.

Reply:
done